normandy/control/static/control/js/components/form_fields/CheckboxField.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const CheckboxField = (props) => {
```

Comment:
I was trying to be consistent with other parts of the codebase, but I too like a function declaration over this style. I'll change it and we'll see what happens when linting lands.